CONTRIBUTING.md

```diff
@@ -8,21 +8,35 @@ Thanks for your interest in contributing to Stencil! :tada:
 Please see our [Contributor Code of Conduct](https://github.com/ionic-team/stencil/blob/main/CODE_OF_CONDUCT.md) for information on our rules of conduct.
 
 
-## Creating an Issue
+## Reporting a Bug
```

Comment:
"Issue" in GitHub nomenclature is a little vague - here, I've split it into 'bug' and 'feature request'